core/src/rpc.rs

```diff
@@ -589,24 +608,8 @@ pub trait RpcSol {
     #[rpc(meta, name = "getFeeRateGovernor")]
     fn get_fee_rate_governor(&self, meta: Self::Metadata) -> RpcResponse<RpcFeeRateGovernor>;
 
-    #[rpc(meta, name = "getSignatureConfirmation")]
-    fn get_signature_confirmation(
-        &self,
-        meta: Self::Metadata,
-        signature_str: String,
-        commitment: Option<CommitmentConfig>,
-    ) -> Result<Option<RpcSignatureConfirmation>>;
-
-    #[rpc(meta, name = "getSignatureStatus")]
-    fn get_signature_status(
-        &self,
-        meta: Self::Metadata,
-        signature_str: String,
-        commitment: Option<CommitmentConfig>,
-    ) -> Result<Option<transaction::Result<()>>>;
-
     #[rpc(meta, name = "getSignatureStatuses")]
-    fn get_signature_statuses(
+    fn get_signature_statuses_with_commitment(
```

Comment:
I only intended this name change in rpc_client. This is actually not consistent with the rest of the rpc methods. Will revert as part of transaction-history work.